2023-05-24 14:28:58,900 [INFO    ] Скрипт запущен: 2023-05-24 14:28:58.899972
2023-05-24 14:28:58,934 [INFO    ] Connected (version 2.0, client OpenSSH_7.4p1)
2023-05-24 14:28:58,983 [INFO    ] Authentication (publickey) failed.
2023-05-24 14:28:58,987 [INFO    ] Connected (version 2.0, client OpenSSH_7.4p1)
2023-05-24 14:28:59,038 [INFO    ] Authentication (password) successful!
2023-05-24 14:28:59,067 [INFO    ] connected to host 92.53.64.66 base preprod_nedvizhka_db
2023-05-24 14:28:59,183 [INFO    ] yandex token valid
2023-05-24 14:29:01,253 [INFO    ] Успешно загружено 1 файлов с личного диска: ['циан 17-05-23 (без дублей).csv'] для cian
2023-05-24 14:29:01,253 [INFO    ] ready to process files [], ['циан 17-05-23 (без дублей).csv']
2023-05-24 14:29:01,253 [INFO    ] обработка файла циан 17-05-23 (без дублей).csv для cian
2023-05-24 14:29:02,369 [INFO    ] обработка district_id:
2023-05-24 14:29:02,372 [INFO    ] districts_update:   0%|          | 0/121065 [00:00<?, ?it/s]
2023-05-24 14:29:17,372 [INFO    ] districts_update:  12%|#2        | 14781/121065 [00:15<01:47, 985.39it/s]
2023-05-24 14:29:32,373 [INFO    ] districts_update:  24%|##4       | 29628/121065 [00:30<01:32, 987.94it/s]
2023-05-24 14:29:42,437 [INFO    ] districts_update:  33%|###2      | 39544/121065 [00:40<01:22, 987.94it/s]
2023-05-24 14:29:47,374 [INFO    ] districts_update:  37%|###6      | 44395/121065 [00:45<01:17, 986.33it/s]
2023-05-24 14:30:02,374 [INFO    ] districts_update:  49%|####9     | 59534/121065 [01:00<01:01, 995.38it/s]
2023-05-24 14:30:12,472 [INFO    ] districts_update:  57%|#####7    | 69417/121065 [01:10<00:51, 995.38it/s]
2023-05-24 14:30:17,374 [INFO    ] districts_update:  61%|######1   | 74425/121065 [01:15<00:46, 994.41it/s]
2023-05-24 14:30:32,375 [INFO    ] districts_update:  74%|#######3  | 89338/121065 [01:30<00:31, 994.34it/s]
2023-05-24 14:30:42,481 [INFO    ] districts_update:  82%|########2 | 99415/121065 [01:40<00:21, 994.34it/s]
2023-05-24 14:30:47,375 [INFO    ] districts_update:  86%|########6 | 104241/121065 [01:45<00:16, 994.06it/s]
2023-05-24 14:31:02,376 [INFO    ] districts_update:  99%|#########8| 119299/121065 [02:00<00:01, 997.17it/s]
2023-05-24 14:31:04,214 [INFO    ] districts_update: 100%|##########| 121065/121065 [02:01<00:00, 993.62it/s]
2023-05-24 14:31:04,215 [INFO    ] district add from realty
2023-05-24 14:31:04,918 [INFO    ] Успешно обработан файл циан 17-05-23 (без дублей).csv для realty
2023-05-24 14:31:04,926 [INFO    ] Выгрузка в таблицу realty обработанного файла из cian: циан 17-05-23 (без дублей).csv
2023-05-24 14:31:05,015 [INFO    ] Временная таблица очищена
2023-05-24 14:31:10,149 [INFO    ] Загрузка новых объявлений в таблицу
2023-05-24 14:31:10,288 [INFO    ] 37964 существующих и 83101 новых объявлений
2023-05-24 14:31:22,092 [INFO    ] обновлена таблица realty
2023-05-24 14:31:23,415 [INFO    ] обновлены районы в таблице realty
2023-05-24 14:31:23,550 [INFO    ] Обновление существующих данных realty завершено
2023-05-24 14:31:35,067 [INFO    ] из новых объявлений удалось найти по адресу 38525, запрос в ddt для 44576
2023-05-24 14:31:35,067 [INFO    ] обновление районов для найденных в realty по адресу новых объявлений:
2023-05-24 14:31:35,308 [INFO    ] обнаружено 1382 районов для 1382 новых объявлений, найденных по адресу из 9313
2023-05-24 14:31:35,337 [INFO    ] 1382 районов осталось незаполнено
2023-05-24 14:31:35,337 [INFO    ] переход к обновлению оставшихся (не найденных по адресу) 44576 новых объявлений для cian
2023-05-24 14:31:35,350 [INFO    ] попытка загрузить данные dadata из /var/www/yadisk_loader/saved_data_csv/cian_dadata_request_2023_05_17.csv
2023-05-24 14:31:35,526 [INFO    ] удалось загрузить исторические данные dadata
2023-05-24 14:31:35,545 [INFO    ] количество запросов к dadata составит: 10369
2023-05-24 14:31:38,560 [ERROR   ] новые объявления не добавлены
2023-05-24 14:31:38,560 [ERROR   ] Traceback (most recent call last):
  File "/var/www/yadisk_loader/main_utils.py", line 387, in load_and_update_realty_db
    df_dadata_houses = dadata_request(df_realty_new, fdate, source)
  File "/var/www/yadisk_loader/dadata_update_utils.py", line 83, in dadata_request
    for i, row in tqdm(df.drop_duplicates(subset='addr').iterrows(), count_zapros.shape[0],
  File "/var/www/yadisk_loader/yadisk_venv/lib/python3.9/site-packages/tqdm/std.py", line 1095, in __init__
    self.refresh(lock_args=self.lock_args)
  File "/var/www/yadisk_loader/yadisk_venv/lib/python3.9/site-packages/tqdm/std.py", line 1344, in refresh
    self.display()
  File "/var/www/yadisk_loader/yadisk_venv/lib/python3.9/site-packages/tqdm/std.py", line 1492, in display
    self.sp(self.__str__() if msg is None else msg)
  File "/var/www/yadisk_loader/yadisk_venv/lib/python3.9/site-packages/tqdm/std.py", line 1148, in __str__
    return self.format_meter(**self.format_dict)
  File "/var/www/yadisk_loader/yadisk_venv/lib/python3.9/site-packages/tqdm/std.py", line 472, in format_meter
    bool_prefix_colon_already = (prefix[-2:] == ": ")
TypeError: 'int' object is not subscriptable

2023-05-24 14:31:38,624 [ERROR   ] Ошибка: ['error_loading_into_realty']
2023-05-24 14:31:38,624 [ERROR   ] Не удалось добавить данные в таблицу realty. Перезапуск скрипта...
2023-05-24 14:31:48,667 [INFO    ] Connected (version 2.0, client OpenSSH_7.4p1)
2023-05-24 14:31:48,715 [INFO    ] Authentication (publickey) failed.
2023-05-24 14:31:48,718 [INFO    ] Connected (version 2.0, client OpenSSH_7.4p1)
2023-05-24 14:31:48,769 [INFO    ] Authentication (password) successful!
2023-05-24 14:31:48,770 [INFO    ] connected to host 92.53.64.66 base preprod_nedvizhka_db
2023-05-24 14:31:48,770 [INFO    ] подключение к базе восстановлено
2023-05-24 14:31:48,870 [INFO    ] нет новых данных для загрузки
